modules/data-streams/src/main/java/org/elasticsearch/datastreams/DataStreamsPlugin.java

```diff
@@ -281,6 +301,7 @@ public List<RestHandler> getRestHandlers(
             handlers.add(new RestPutDataStreamOptionsAction());
             handlers.add(new RestDeleteDataStreamOptionsAction());
         }
+        handlers.add(new RestReindexDataStreamAction());
```

Comment:
Do we want to add the rest handler already?

Reply:
No, I had accidentally left that in (I've been using it locally)! It's gone now.